test/functional/legacy/search_mbyte_spec.lua

```diff
@@ -0,0 +1,31 @@
+-- Test for search('multi-byte char', 'bce')
```

Comment:
these comments can go in the `describe()` or `it()` descriptions instead. I'll fix this when merging so we don't hold up this PR any longer.